README.md

```diff
@@ -111,3 +112,56 @@ interval=60
 
 Example output of the script:\
 ```10:55 Grocery shopping```
+
+### How to use the `skip` flag to scroll events
+
+Edit the polybar configuration creating two modules:
+
+```ini
+[module/agenda-ipc]
+type = custom/ipc
```

Comment:
I'm not familiar with the ipc module, hooks etc. Could you please refer me to the relevant documentation?

Reply:
sure:
* Wiki: https://github.com/polybar/polybar/wiki/Module:-ipc
* Related issue: https://github.com/polybar/polybar/issues/786

Reply:
Thank you